scripts/sync-workflow.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const lr = require('load-runner')();
+const request = require('request');
+
+const requestBodyUtils = require('../util/sync_request_bodies');
+
+const argv = require('yargs')
+      .reset()
```

Comment:
Hmm, good question. ESLint doesn't seem to be complaining about it. Emacs is indenting it out this far for me, but if I set it to basically any other indent, it's also fine with it.

Looking into it a little further, there's an option for the ESLint `indent` rule called [MemberExpression](http://eslint.org/docs/rules/indent#memberexpression), but even that says "Any indentation is permitted in variable declarations and assignments", which is the case here.

Would you prefer two spaces here instead of the alignment that it's currently doing? I can easily change it to that, and get Emacs to behave. :)